spotify/main.py

```diff
@@ -175,8 +201,8 @@ def main():
         logger.error("Open Podcast API is not up. Quitting")
         sys.exit(1)
 
-    start = dt.datetime.now() - dt.timedelta(days=1)
```

Comment:
Since the default `start` is today - 4 days, we change the date range here. Before it was 1 day, now it would be 3 days. Is that expected and still okay?

Reply:
the metadata api ignores dates at all, so we can also use our default values here.

Reply:
Great, thanks! That's what I suspected, but I wanted to make 100% sure.